### PR TITLE
Create Page for Default Uploaders

### PR DESCRIPTION
**Pull request by tayron1**

### CODE REVIEW

File: pages/read/sxcu.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            text: 'Go to the <a href="https://getsharex.com" class="link">ShareX Website</a>.'
+        }, {
+            id: 2,
+            text: 'Download and open the installer.'
+        }, {
+            id: 3,
+            text: 'Go through the Installer.'
+        }, {
+            id: 4,
+            text: 'Open the Intigrations Tab on the Tixte Website and download the Standard Configuration for your desired Domain.',
```
Comment (Luna-devv) [inline pages/read/sxcu.jsx:21]: Would be nice to link the `Intigrations Tab` here

File: pages/read/sxcu.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            subs: ['Note that you will have to register a Domain and <a href="/read/domains" class="link">set it up with Tixte or set up a Subdomain</a>']
```
Comment (Luna-devv) [inline pages/read/sxcu.jsx:22]: why `or`? You have to do both

File: pages/read/sxcu.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        },
+    { 
+        id: 5,
+        text: 'Open the Configuaration (the Downloaded File) and confirm the Confirmation Box made by ShareX.'
```
Comment (Luna-devv) [inline pages/read/sxcu.jsx:26]: Would rather put `Open the downloaded configuration file and confirm the pop-up from shareX.`

File: pages/read/sxcu.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        section: 'how',
+        list: [{
+            id: 1,
+            text: 'Go to the <a href="https://getsharex.com" class="link">ShareX Website</a>.'
+        }, {
+            id: 2,
+            text: 'Download and open the installer.'
+        }, {
+            id: 3,
+            text: 'Go through the Installer.'
```
Comment (Luna-devv) [inline pages/read/sxcu.jsx:18]: Why is `Installer` uppercase here when it was lowercase in L#15

Comment (tayron1) [reply pages/read/sxcu.jsx:18]: Fixed

**Pull request merged**